src/cli.rs

```diff
@@ -3,98 +3,100 @@ use crate::file::{
 #[command(after_help = COPYRIGHT)]
-/// file handling
-pub enum FileConfig {
-    /// copy file into image
-    CopyToImage {
-        /// multiple [in-file-path,out-partition:out-file-path]: input file, output file partition and output file
+/// copy files to or from a firmware image
+pub enum FileCopy {
+    /// copy files into image
+    ToImage {
+        /// vector of copy triples in the format [in-file-path,out-partition:out-file-path]
         #[clap(short = 'f', long = "files", value_parser = clap::value_parser!(FileCopyToParams), required(true))]
         file_copy_params: Vec<FileCopyToParams>,
         /// path to wic image file
         #[arg(short = 'i', long = "image")]
-        image: std::path::PathBuf,
+        image: PathBuf,
         /// optional: generate bmap file
         #[arg(short = 'b', long = "generate-bmap-file")]
         generate_bmap: bool,
         /// optional: pack image [xz, bzip2, gzip]
         #[arg(short = 'p', long = "pack-image", value_enum)]
         compress_image: Option<Compression>,
     },
-    /// copy file from image
-    CopyFromImage {
-        /// multiple [in-partition:in-file-path,out-file-path]: input file partition, input file and output file
+    /// copy files from image
+    FromImage {
+        /// vector of copy triples in the format [in-partition:in-file-path,out-file-path]
         #[clap(short = 'f', long = "files", value_parser = clap::value_parser!(FileCopyFromParams), required(true))]
         file_copy_params: Vec<FileCopyFromParams>,
         /// path to wic image file
         #[arg(short = 'i', long = "image")]
-        image: std::path::PathBuf,
+        image: PathBuf,
     },
 }
 
 #[derive(Parser, Debug)]
 #[command(after_help = COPYRIGHT)]
-/// pre-configure device identity settings
+/// configure Azure IoT identity settings
 pub enum IdentityConfig {
-    /// set general config.toml file
+    /// configure identity settings of a standard iot or iotedge device (no transparent gateway nor iot leaf device)
     SetConfig {
         /// path to config.toml file
         #[arg(short = 'c', long = "config")]
-        config: std::path::PathBuf,
+        config: PathBuf,
         /// optional: path to extra DPS payload file
         #[arg(short = 'e', long = "extra-dps-payload")]
-        payload: Option<std::path::PathBuf>,
+        payload: Option<PathBuf>,
         /// path to wic image file
         #[arg(short = 'i', long = "image")]
-        image: std::path::PathBuf,
+        image: PathBuf,
         /// optional: generate bmap file
         #[arg(short = 'b', long = "generate-bmap-file")]
         generate_bmap: bool,
         /// optional: pack image [xz, bzip2, gzip]
         #[arg(short = 'p', long = "pack-image", value_enum)]
         compress_image: Option<Compression>,
     },
-    /// set transparent gateway config.toml file and additional certificates and keys
+    /// EXPERIMENTAL: set transparent gateway config.toml file and additional certificates and keys
```

Comment:
why is it marked "EXPERMENTAL"?

Reply:
since it wasn't tested for a long time. further it is nothing we currently support. i want to avoid that somebody is talking to us about this kind of architecture. i was even thinking about removing it here. what is your opinion?